.github/workflows/pullrequest.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       options: --platform linux/amd64
     steps:
     - uses: actions/checkout@v2
-    - uses: bufbuild/buf-setup-action@v1
+    - uses: bufbuild/buf-setup-action@v1.8.0
```

Comment:
Do we need to pin this? If yes, let's do 1.9.0

Reply:
no I just forgot to undo that